core/templates/dev/head/dashboard/DashboardServices.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Standalone services for the creator dashboard page.
+ */
+
+// Service for sorting the explorations based on different parameters.
+oppia.factory('sortExplorationsService', [
+  'utilsService', function(utilsService) {
+    var SORT_BY_KEYS = [
```

Comment:
I think a better way to deal with the constant is to put this as a public constant, and make it a dictionary of strings, like
```
var SORT_BY_KEYS = {
  TITLE: 'title',
  ...
}
```
This way, when choosing the sort type, you can reference this dictionary, instead of having to use the strings directly.